core/src/test/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisherTests.java

```diff
@@ -137,6 +137,28 @@ public void unknownFailureExceptionIsIgnored() {
 		verifyZeroInteractions(appPublisher);
 	}
 
+	@Test(expected = IllegalArgumentException.class)
+	public void defaultAuthenticationFailureEventClassSetNullThen() {
+		publisher = new DefaultAuthenticationEventPublisher();
+		publisher.setDefaultAuthenticationFailureEvent(null);
+	}
+
+	@Test
+	public void defaultAuthenticationFailureEventIsPublished() {
+		publisher = new DefaultAuthenticationEventPublisher();
+		Properties p = new Properties();
```

Comment:
Are these lines necessary that set up the additional exception mappings? I'm thinking that lines 149 thru 152 can be removed.